diff --git a/platform/java/Makefile b/platform/java/Makefile
index 2eaf448..333618f 100644
--- a/mupdf/platform/java/Makefile
+++ b/mupdf/platform/java/Makefile
@@ -76,7 +76,7 @@ jar: $(MUPDF_JAR)
 MUPDF_CORE := ../../$(OUT)/libmupdf.a ../../$(OUT)/libmupdf-third.a
 ifeq "$(OS)" "Linux"
 $(MUPDF_CORE) : .FORCE
-	$(MAKE) -C ../.. OUT=$(OUT) XCFLAGS=-fPIC build=$(build) USE_SYSTEM_FREETYPE=yes USE_SYSTEM_ZLIB=yes $(CONFIGFLAGS) libs
+	$(MAKE) -C ../.. OUT=$(OUT) XCFLAGS=-fPIC build=$(build) USE_SYSTEM_LIBJPEG=no USE_SYSTEM_OPENJPEG=no USE_SYSTEM_FREETYPE=yes USE_SYSTEM_ZLIB=yes $(CONFIGFLAGS) libs
 else
 $(MUPDF_CORE) : .FORCE
 	$(MAKE) -C ../.. OUT=$(OUT) XCFLAGS=-fPIC build=$(build) $(CONFIGFLAGS) libs
@@ -86,7 +86,7 @@ LIBRARY_JAVA_SOURCES := $(sort $(wildcard src/com/artifex/mupdf/fitz/*.java))
 LIBRARY_JAVA_OBJECTS := $(subst src/,../../$(OUT)/,$(LIBRARY_JAVA_SOURCES:%.java=%.class))
 LIBRARY_JAVA_CLASSES := $(subst src/com/artifex/mupdf/fitz/,com.artifex.mupdf.fitz.,$(LIBRARY_JAVA_SOURCES:%.java=%))
 
-JAVAC_FLAGS := $(JAVAC_XFLAGS) -source 1.7 -target 1.7
+JAVAC_FLAGS := $(JAVAC_XFLAGS)
 
 $(LIBRARY_JAVA_OBJECTS) : $(LIBRARY_JAVA_SOURCES)
 	mkdir -p ../../$(OUT)
